src/dev/build/tasks/clean_tasks.js

```diff
@@ -29,39 +29,36 @@ export const CleanTask = {
   description: 'Cleaning artifacts from previous builds',
 
   async run(config, log) {
-    await deleteAll([
```

Comment:
@clintandrewhall please disable the prettier plugin in your editor, it doesn't know which files to edit and which to leave along. Just use the eslint plugin, we use eslint-plugin-prettier so that eslint will apply its own fixes and prettiers as configured for the repo.

Reply:
yeah, @tylersmalley clued me in to that in Slack. 👍